app/service/rest.py

```diff
@@ -506,6 +506,21 @@ def get_all_notifications_for_service(service_id):
     )
 
 
+@service_blueprint.route("/<uuid:service_id>/notifications/count", methods=["GET"])
+def count_notifications_for_service(service_id):
+    data = notifications_filter_schema.load(request.args)
+
+    multidict = MultiDict(data)
+
+    notification_count = fact_billing_dao.get_count_of_notifications_sent(
+        service_id=service_id,
+        template_types=multidict.getlist("template_type"),
+        limit_days=data.get("limit_days"),
```

Comment:
do we want to default these two query params to sensible defaults? or error if they're not supplied?

maybe limit_days=7, template_type=['sms', 'email', 'letter'] but i'm open if you have other thoughts (or if the schema already does this by default and i didnt realise)

Reply:
https://github.com/alphagov/notifications-api/pull/4160/commits/db48dee691df1fc0d41abd2e5aa2e14f839e1bfe